pkg/reconciler/reconciler.go

```diff
@@ -146,6 +148,7 @@ func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
 // Option is a function that configures the helm Reconciler.
```

Comment:
@joelanford I checked that many options are not used in the main.go. Following the questions: 
 When and how it can be setup? What  is the user case?